### PR TITLE
Add ASTE recipe for the latest ASTE release

### PR DESCRIPTION
**Pull request by davidscn**
... we could also test the validity of the installation by running `ctest`. Do you usually do this during the installation?

### CODE REVIEW

**Comment by MakisH** [comment]
Looking at the code, it looks good. Thanks for directly contributing!

> we could also test the validity of the installation by running ctest. Do you usually do this during the installation?

I usually don't do that here, since the build time of the VM is already quite long. It is not meant as an environment for the system tests.

However, if you think this is ready, you can trigger a build of the VM and try it yourself. You can trigger that by converting the PR to draft and then marking it ready for review (or directly from the Actions menu, at least after the first time).

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ This box is based on the [generic/ubuntu2004](https://github.com/lavabit/robox/t
 - SU2 6.0.0 and the SU2-preCICE adapter (master)
 - code_aster 14.6 and the code_aster-preCICE adapter (master)
 - DUNE 2.8 and the experimental DUNE-preCICE adapter (master)
+- ASTE (vX.X)
```
Comment (davidscn) [inline README.md:61]:
```suggestion
- ASTE (v3.0.0)
```

File: README.md [hunk 2]
```diff
@@ -58,6 +58,7 @@ This box is based on the [generic/ubuntu2004](https://github.com/lavabit/robox/t
 - SU2 6.0.0 and the SU2-preCICE adapter (master)
 - code_aster 14.6 and the code_aster-preCICE adapter (master)
 - DUNE 2.8 and the experimental DUNE-preCICE adapter (master)
+- ASTE (v3.0.0)
```
Comment (davidscn) [inline README.md:61]:
```suggestion
- ASTE (master)
```

**Comment by MakisH** [comment]
Building currently seems to fail at downloading code-aster (unrelated), due to an SSL certificate issue. I will check again when we actually need to release the VM and potentially work around it.

**Review by MakisH — APPROVED** [review]
Building the vm now also worked, so we can already merge this, unless you still want to try something additional.

**Pull request merged**